package.json

```diff
@@ -1,18 +1,27 @@
 {
```

Comment:
I wonder if we have integration tests that run this inside a browser?
I see that the `README.md` says `_Disabled during v2.0.0-beta_`. Is this correct?

Reply:
We were running all of our tests in a virtual empty javascript context. But had to drop it because it wouldn't load `wasm` files and that configuration was design for `commonjs`.
I'll research adding an integration test with a browser. `karma` seems to be deprecated now.

I'll update the Readme

Reply:
I've used `puppeteer` before for CI testing, and IIRC, you can load an `.html` in-memory and just query the elements there.